test/Rx/Functional/Operator/IsEmptyTest.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+namespace Rx\Functional\Operator;
+
+use Rx\Functional\FunctionalTestCase;
+
+class IsEmptyTest extends FunctionalTestCase
+{
+
+    /**
+     * @test
+     */
+    public function should_return_true_if_source_is_empty()
+    {
+        $xs = $this->createHotObservable([
+            onCompleted(300)
```

Comment:
Since this is a hot observable, can you add an onNext before 200, so we can verify that nothing emits before it's suppose to?